components/contact-details.tsx

```diff
@@ -29,6 +29,7 @@ type DescriptionItemProps = {
   value?: string;
   withCopyButton?: boolean;
   withOpenMapButton?: boolean;
+  isLink?: boolean;
```

Comment:
```suggestion
  hasOverflowScroll?: boolean;
```

Please use a more general purpose prop name.